### ISSUE TITLE
[Feature Request] Backup to ExtSD

### ISSUE THREAD

**Issue by DavidFirefox** [federicoiosue/Omni-Notes#651]
Hello,
Thanks a lot for this great software.
I think put an option for backup to ExtSD will be a great idea.
Or the possibility to choose the backup directory.
I think do backup on ExtSD is better, because if the phone is broken we can easily read the ExtSD Card.

Is it lot of work to do it ?

Thanks in advance
David

**Comment by federicoiosue** [comment]
You're right, the current mechanism has been implemented when dealing with external SD was a problem but now nothing is blocking this feature to be developed.

**Comment by LuGO0** [comment]
what if there was no SD in place??
